apps/docs/content/guides/auth/social-login/auth-workos.mdx

```diff
@@ -1,172 +1,129 @@
-	}
+  if (data.url) {
+    redirect(data.url) // use the redirect API for your server or framework
+  } 
 }
 ```
-
-Refer to the [WorkOS Documentation](https://workos.com/docs/reference/sso/authorize/) to learn more about the different methods.
-
-</TabPanel>
-</Tabs>
-
-<OAuthPkceFlow />
-
-<Tabs
-  scrollable
-  size="small"
-  type="underlined"
-  defaultActiveId="js"
-  queryGroup="language"
->
-<TabPanel id="js" label="JavaScript">
-
-When your user signs out, call [signOut()](/docs/reference/javascript/auth-signout) to remove them from the browser session and any objects from localStorage:
-
-```js
-async function signOut() {
-  const { error } = await supabase.auth.signOut()
+**Note**: In the above code, you can find your `connection_id` in the WorkOS dashboard under the Organizations tab. Select your organization and then click View connection.
+
+Within your specified callback URL, you'll exchange the code for a logged in user profile: 
+
+```javascript auth/v1/callback/route.ts
+import { NextResponse } from 'next/server'
+import { createClient } from '@/utils/supabase/server'
+
+export async function GET(request: Request) {
+  const { searchParams, origin } = new URL(request.url)
+  const code = searchParams.get('code')
+  // if "next" is in param, use it as the redirect URL
+  const next = searchParams.get('next') ?? '/'
+
+  if (code) {
+    const supabase = await createClient()
+    const { error } = await supabase.auth.exchangeCodeForSession(code)
+    if (!error) {
+      const forwardedHost = request.headers.get('x-forwarded-host') // original origin before load balancer
+      const isLocalEnv = process.env.NODE_ENV === 'development'
+      if (isLocalEnv) {
+        // we can be sure that there is no load balancer in between, so no need to watch for X-Forwarded-Host
+        return NextResponse.redirect(`${origin}${next}`)
+      } else if (forwardedHost) {
+        return NextResponse.redirect(`https://${forwardedHost}${next}`)
+      } else {
+        return NextResponse.redirect(`${origin}${next}`)
+      }
+    }
+  }
+
+  // return the user to an error page with instructions
+  return NextResponse.redirect(`${origin}/auth/auth-code-error`)
 }
-```
-
-</TabPanel>
-<TabPanel id="flutter" label="Flutter">
 
-When your user signs out, call [signOut()](/docs/reference/dart/auth-signout) to remove them from the browser session and any objects from localStorage:
-
-```dart
-Future<void> signOut() async {
-  await supabase.auth.signOut();
-}
 ```
 
-</TabPanel>
-<TabPanel id="kotlin" label="Kotlin">
+## Option 2: Using AuthKit
 
-When your user signs out, call [signOut()](/docs/reference/kotlin/auth-signout) to remove them from the browser session and any objects from localStorage:
-
-```kotlin
-suspend fun signOut() {
-	supabase.auth.signOut()
-}
-```
+### Step 1. Install and configure AuthKit
 
-</TabPanel>
-</Tabs>
+Visit [https://authkit.com](https://authkit.com) to get up and running with AuthKit, the world's best login box. 
```

Comment:
I think we should link to the Quick Start directly rather than the marketing page.

Reply:
Good call

Reply:
Done in [54d86d0](https://github.com/zackproser-workos/supabase/pull/1/commits/54d86d0d34e45c7f55496e723c2fa235fc0cf083)